website/blog/2020-01-10-7.8.0.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title:  "7.8.0 Released: ECMAScript 2020, .mjs configuration files and @babel/cli improvements"
+author: Nicolò Ribaudo
+authorURL: https://twitter.com/NicoloRibaudo
+date:   2020-01-10 20:00:00
+categories: announcements
+share_text: "Babel 7.8.0 Released"
+---
+
+This is the first release of the year! :tada: 
+
+Babel 7.8.0 supports the new [ECMAScript 2020](https://github.com/tc39/proposals/blob/master/finished-proposals.md) features by default: you don't need to enable individual plugins for nullish coalescing (`??`), optional chaining (`?.`) and dynamic `import()` anymore with preset-env.
+
+We also finished unifying our different configuration formats, a process that we started in the [7.7.0 release](https://babeljs.io/blog/2019/11/05/7.7.0#new-configuration-file-extensions-10501-https-githubcom-babel-babel-pull-10501-10599-https-githubcom-babel-babel-pull-10599).
```

Comment:
we wanted to use another word other than 'unify' in this case right?

Reply:
Oh right

Reply:
Does this look good?

```suggestion
We also finished aligning our different configuration files with the formats natively supported by Node.js, a process that we started in the [7.7.0 release](https://babeljs.io/blog/2019/11/05/7.7.0#new-configuration-file-extensions-10501-https-githubcom-babel-babel-pull-10501-10599-https-githubcom-babel-babel-pull-10599).
```